The osx failure is:

```
make[1]: *** No rule to make target `/Users/bion/vcpkg/installed/x64-osx/debug/lib/libQt6OpenGLWidgets.a', needed by `../lib/libqwt.6.2.0.dylib'.  Stop.
make: *** [sub-src-all-ordered] Error 2
```

I observe there is no `libQt6OpenGLWidgets.a` but there is `installed/x64-osx/debug/lib/libQt6OpenGLWidgets_debug.a` Is that ringing any bells in terms of obvious mixups?

--- a/ports/qwt-qt6/disable-examples-playground-tests.patch
+++ b/ports/qwt-qt6/disable-examples-playground-tests.patch
@@ -1,0 +1,30 @@
+diff --git a/qwtconfig.pri b/qwtconfig.pri
+index 0b054e0..d922aff 100644
+--- a/qwtconfig.pri
++++ b/qwtconfig.pri
+@@ -137,7 +137,7 @@ win32 {
+ # Otherwise you have to build them from the examples directory.
+ ######################################################################
+ 
+-QWT_CONFIG     += QwtExamples
++#QWT_CONFIG     += QwtExamples
+ 
+ ######################################################################
+ # The playground is primarily intended for the Qwt development
+@@ -148,14 +148,14 @@ QWT_CONFIG     += QwtExamples
+ # Otherwise you have to build them from the playground directory.
+ ######################################################################
+ 
+-QWT_CONFIG     += QwtPlayground
++#QWT_CONFIG     += QwtPlayground
+ 
+ ######################################################################
+ # If you want to auto build the tests, enable the line below
+ # Otherwise you have to build them from the tests directory.
+ ######################################################################
+ 
+-QWT_CONFIG     += QwtTests
++#QWT_CONFIG     += QwtTests
+ 
+ ######################################################################
+ # When Qt has been built as framework qmake wants

--- a/ports/qwt-qt6/portfile.cmake
+++ b/ports/qwt-qt6/portfile.cmake
@@ -1,0 +1,37 @@
+vcpkg_from_sourceforge(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO qwt/qwt
+    REF 6.2.0
+    FILENAME "qwt-6.2.0.zip"
+    SHA512 a3946c6e23481b5a2193819a1c1298db5a069d514ca60de54accb3a249403f5acd778172ae6fae24fae252767b1e58deba524de6225462f1bafd7c947996aae9
+    PATCHES disable-examples-playground-tests.patch
+)
+
+vcpkg_qmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        "CONFIG+=${VCPKG_LIBRARY_LINKAGE}"
+)
+
+if (VCPKG_TARGET_IS_WINDOWS)
+    vcpkg_install_qmake(
+        RELEASE_TARGETS sub-src-release_ordered
+        DEBUG_TARGETS sub-src-debug_ordered
+    )
+else ()
+    vcpkg_install_qmake(
+        RELEASE_TARGETS sub-src-all-ordered
+        DEBUG_TARGETS sub-src-all-ordered
+    )
+endif()
+
+if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
+    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/bin" "${CURRENT_PACKAGES_DIR}/debug/bin")
+endif()
+
+#Install the header files
+file(GLOB HEADER_FILES "${SOURCE_PATH}/src/*.h")
+file(INSTALL ${HEADER_FILES} DESTINATION "${CURRENT_PACKAGES_DIR}/include/${PORT}")
+
+# Handle copyright
+file(INSTALL "${SOURCE_PATH}/COPYING" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)

--- a/ports/qwt-qt6/vcpkg.json
+++ b/ports/qwt-qt6/vcpkg.json
@@ -1,0 +1,30 @@
+{
+  "name": "qwt-qt6",
+  "version-semver": "6.2.0",
+  "description": "qt widgets library for technical applications",
+  "homepage": "https://sourceforge.net/projects/qwt",
+  "dependencies": [
+    {
+      "name": "qtbase",
+      "default-features": false,
+      "features": [
+        "concurrent",
+        "gui",
+        "widgets"
+      ]
+    },
+    "qtsvg",
+    {
+      "name": "qttools",
+      "default-features": false,
+      "features": [
+        "designer"
+      ]
+    },
+    {
+      "name": "vcpkg-qmake",
+      "host": true,
+      "default-features": false
+    }
+  ]
+}

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -1231,6 +1231,7 @@ quickfix:x64-windows-static-md=fail
 quickfix:x64-windows=fail
 quickfix:x86-windows=fail
 qwt:x64-osx=fail
+qwt-qt6:x64-osx=fail
 qwtw:x64-windows=fail
 rabit:x64-osx=fail
 ragel:arm-uwp=fail

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5840,6 +5840,10 @@
       "baseline": "6.2.0",
       "port-version": 0
     },
+    "qwt-qt6": {
+      "baseline": "6.2.0",
+      "port-version": 0
+    },
     "qwtw": {
       "baseline": "3.1.0",
       "port-version": 1

--- a/versions/q-/qwt-qt6.json
+++ b/versions/q-/qwt-qt6.json
@@ -1,0 +1,9 @@
+{
+  "versions": [
+    {
+      "git-tree": "96c2132cd5810b4a9950650768fb759738d3b929",
+      "version-semver": "6.2.0",
+      "port-version": 0
+    }
+  ]
+}

